backend/migrations/20220926065450-create-user-follow.js

```diff
@@ -14,6 +14,9 @@ module.exports = {
       follower_id: {
         type: Sequelize.INTEGER,
       },
+      isFollowed: {
+        type: Sequelize.BOOLEAN,
+      },
```

Comment:
- avoid modifying migration files already merged in the dev branch (very important)
- instead generate a new migration [see reference here](https://github.com/escbooster12/practice-expressjs-sequelize/tree/develop/migrations)